presto-main/src/test/java/com/facebook/presto/type/TestArrayOperators.java

```diff
@@ -955,6 +955,8 @@ public void testDistinct()
                 "ARRAY_DISTINCT(ARRAY [2.330, 1.900, 2.330])",
                 new ArrayType(createDecimalType(4, 3)),
                 ImmutableList.of(decimal("2.330"), decimal("1.900")));
+
+        assertCachedInstanceHasBoundedRetainedSize("ARRAY_DISTINCT(ARRAY[2, 3, 4, 1, 2])");
```

Comment:
You need to make sure your `assertCachedInstanceHasBoundedRetainedSize` tests all implementations. Specifically, here, you need to cover both `array(bigint)` and something else.